src/components/field.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+
+import styles from "./field.module.scss"
+import classNames from "classnames"
+
+
+interface Props {
```

Comment:
Could you call this `FieldProps`? Doing this means we can export this without worrying about name clashes for things like testing and previews.